java/src/main/resources/template.properties

```diff
@@ -459,3 +459,11 @@ text.bigquery.output.dataset=
 text.bigquery.output.table=
 text.bigquery.output.mode=
 text.bigquery.temp.bucket=
+
+# PubSubLite To GCS
+pubsublite.to.gcs.input.subscription.url=
+pubsublite.to.gcs.processing.time.seconds=
+pubsublite.to.gcs.write.mode=append
+pubsublite.to.gcs.output.format=json
+pubsublite.to.gcs.output.location=
+pubsublite.to.gcs.checkpoint.location
```

Comment:
`pubsublite.to.gcs.timeout.ms` is missing. This parameter can be optional too, with a default value here